DrupalExtended/ruleset.xml

```diff
@@ -0,0 +1,164 @@
+  <description>An extension of Drupal coding standards.</description>
+
+  <rule ref="vendor/drupal/coder/coder_sniffer/Drupal">
+    <!-- Exclude in favor of native typehints. See: https://www.drupal.org/project/coding_standards/issues/3238192 -->
+    <exclude name="Drupal.Commenting.VariableComment.MissingVar"/>
+    <!-- Conflicts with "SlevomatCodingStandard.TypeHints.DeclareStrictTypes". -->
+    <exclude name="Drupal.WhiteSpace.OpenTagNewline.BlankLine"/>
+    <!-- Conflicts with "<?php declare(strict_types = 1)." -->
+    <exclude name="Drupal.Commenting.FileComment.Missing"/>
+  </rule>
+
+  <rule ref="vendor/drupal/coder/coder_sniffer/DrupalPractice"/>
+
+  <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard">
+    <!-- Excluded rules based on what enabled in 1.x branch. -->
+    <exclude name="SlevomatCodingStandard.Arrays.DisallowImplicitArrayCreation"/>
+    <exclude name="SlevomatCodingStandard.Arrays.MultiLineArrayEndBracketPlacement"/>
+    <exclude name="SlevomatCodingStandard.Arrays.SingleLineArrayWhitespace"/>
+    <!-- Same as 'Squiz.Arrays.ArrayDeclaration.NoCommaAfterLast'. -->
+    <exclude name="SlevomatCodingStandard.Arrays.TrailingArrayComma"/>
+    <exclude name="SlevomatCodingStandard.Attributes.AttributeAndTargetSpacing"/>
+    <exclude name="SlevomatCodingStandard.Attributes.AttributesOrder"/>
+    <exclude name="SlevomatCodingStandard.Attributes.DisallowAttributesJoining"/>
+    <exclude name="SlevomatCodingStandard.Attributes.DisallowMultipleAttributesPerLine"/>
+    <exclude name="SlevomatCodingStandard.Attributes.RequireAttributeAfterDocComment"/>
+    <exclude name="SlevomatCodingStandard.Classes.BackedEnumTypeSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassLength"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassMemberSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassStructure"/>
+    <exclude name="SlevomatCodingStandard.Classes.ConstantSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowConstructorPropertyPromotion"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowMultiConstantDefinition"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowMultiPropertyDefinition"/>
+    <exclude name="SlevomatCodingStandard.Classes.EmptyLinesAroundClassBraces"/>
+    <exclude name="SlevomatCodingStandard.Classes.ForbiddenPublicProperty"/>
+    <exclude name="SlevomatCodingStandard.Classes.MethodSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.ParentCallSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.PropertyDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Classes.PropertySpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.RequireAbstractOrFinal"/>
+    <exclude name="SlevomatCodingStandard.Classes.RequireConstructorPropertyPromotion"/>
```

Comment:
Sounds good, couldn't find any issues with it on my project. It seems that it properly handles old-fashion constructors if they are inherited from parent class.